addons/mail/static/src/discuss/core/common/thread_actions.js

```diff
@@ -186,3 +187,23 @@ registerThreadAction("delete-thread", {
     sequenceGroup: 40,
     tags: [ACTION_TAGS.DANGER],
 });
+registerThreadAction("message-replies", {
+    actionPanelComponent: MessageRepliesPanel,
+    condition: false, // Not shown in the action list, gets managed by methods in `env`
+    actionPanelComponentCondition: ({ owner }) => owner.env.repliesMenu,
+    setup: ({ action }) =>
+        useSubEnv({
+            repliesMenu: {
+                open: (message) => {
+                    action.open();
+                    action.definition.actionPanelComponentProps = () => ({ message });
+                },
+                close: () => {
+                    if (action.isActive) {
+                        action.close();
+                    }
```

Comment:
In `Meeting` view, the chat is a panel (`meeting-chat`).
The showing of replies message is made from meeting-chat, so when closing the replies panel we'd expect to come back to the chat.

We have a feature of `keepPrevious` that can be used specifically for `Meeting`, so that it comes back to chat. Or we re-open the `meeting-chat` panel on close of the replies message panel.